src/libraries/System.Runtime.Caching/src/System/Runtime/Caching/MemoryCacheStatistics.cs

```diff
@@ -340,8 +340,14 @@ public void Dispose()
                     GCHandleRef<Timer> timerHandleRef = _timerHandleRef;
                     if (timerHandleRef != null && Interlocked.CompareExchange(ref _timerHandleRef, null, timerHandleRef) == timerHandleRef)
                     {
-                        timerHandleRef.Dispose();
-                        Dbg.Trace("MemoryCacheStats", "Stopped CacheMemoryTimers");
+                        // #64115 and #102666 - It's not safe to dispose timers in our unhandled exception handler. But if we're in the unhandled
```

Comment:
I think we generally avoid archiving bug numbers in code because they are clutter and can be gotten easily from git blame, which gives all kinds of other info as well. This wasn't as easy in old SD/TFS days.